assets/aml-benchmark/components/inference-postprocessor/spec.yaml

```diff
@@ -29,10 +29,16 @@ inputs:
       Key in ground truth dataset that contains ground truth. If ground_truth_dataset is given, then,
       this is required input.
     optional: True
-  prediction_probabilities_dataset:
```

Comment:
@arun-rajora Because model prediction component has made ground truth field optional. In case this component becomes a part of inference, then this will be applicable.